vortex-flatbuffers/flatbuffers/vortex-file/footer.fbs

```diff
@@ -105,7 +105,27 @@ table CompressionSpec {
     scheme: CompressionScheme;
 }
 
+// When encrypting data with AES_GCM the nonce should always be 12 bytes in length (per specification recommendation) and tag 16 bytes.
+// If we were to add AES_CTR we would keep same 12 byte nonce and append a 32 bit counter to it (initialzed at 1) to encrypt data.
+
+// Possible encryption algorithms to use when encrypting files
+enum EncryptionAlgorithm: uint8 {
+    AES_GCM = 0,
+    // Do we need random access to the encrypted bytes? Since our individual blocks are relatively small there should be a minor
+    // difference between slicing the block and decrypting the whole thing, however, maybe there is?
+    // AES_CTR = 1,
```

Comment:
I think for 2MB segments the performance difference could be meaningful for random-access. Basically if we use GCM and we have multi-MB blocks, we lose the benefit of the granular zone maps. Maybe that's just the price you pay?